docs/source/programming_resources/shared/required_materials/Required-Materials.rst

```diff
@@ -47,33 +50,40 @@ tutorials, you will need to have the following materials available:
    * - Required Item(s)
      - Image
 
-   * - Two (2) FIRST-approved* Android devices OR One (1) Control Hub and one (1) FIRST-approved* Android device for the DRIVER STATION
-     - |androidphones|   Or |chandphones|
+   * - A REV Robotics Control Hub to be used as a ROBOT CONTROLLER.
+       Alternately, the Competition Manual does allow teams to instead
+       use a REV Robotics Expansion Hub and a select list of Android
+       phones. See rules R701 and R704 for the full details.
```

Comment:
Rephrase to not point to a specific rule number. Rule numbers may shift year to year.  

Suggest something like:

> Check the Robot Construction rules in the [Competition Manual](https://ftc-resources.firstinspires.org/file/ftc/game/manual) for details.